### DISCUSSION TITLE
Retrive Orders List

### DISCUSSION THREAD

**Discussion by umardraz670** [etsy/open-api#773] [Q&A]
Hi ,
How i can retrive a list of placed orders using REST API v3.
Kindly i tell me , from API V3 api reference i only found api for relating to Shopping , Shop Info , Users , etc but not found any api for getting list of orders and related to orders.

Umar D.
MS Software Engineer

**Comment by Gareth064** [comment]
You want the Receipt endpoints.

Receipt = Order

**Reply by umardraz670** [reply]
Got it , but now i am facing issue with oauth 2.0 using java . do you have any idea about java wiith Etsy ?
